src/main/kotlin/com/autonomousapps/internal/utils/gradleStrings.kt

```diff
@@ -87,13 +82,13 @@ internal fun ResolvedArtifactResult.toCoordinates(): Coordinates {
 /** Returns the [coordinates][Coordinates] of the root of [this][Configuration]. */
 internal fun Configuration.rootCoordinates(): Coordinates = incoming.resolutionResult.root.id
   // For the root, the 'GradleVariantIdentification' is always empty as there is only one root (which we match later)
-  .toCoordinates(GradleVariantIdentification(emptySet(), emptyMap()))
+  .toCoordinates(GradleVariantIdentification(setOf("ROOT"), emptyMap()))
```

Comment:
In some places, the ROOT node is part of the graph/classpath although dependencies never point at it. For the root, we do not know which "variant" it is – e.g. are we looking at the "main" or the "test-fixtures" root? Since there might be different nodes in the graph where dependencies point back at a variant of the "root" the root itself should not be confused with any of these. Hence we give it a special capability "ROOT" here. Otherwise, the plugin thinks that the ROOT is always equal to the "main" variant.

This ROOT could probably be removed in some places where we do analysis. It is not relevant as a target of a dependency, when the nodes are compared to compute advices.